tests/variable/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -26,5 +58,38 @@ $_4site = 'not yet';
 $täyte = 'mansikka';
 $bar = &$foo;
 $bar = "My name is $bar";
+$id = $element->$id_field;
+$id = $element[$id_field];
+$var = <<<EOT
+My name is "$name". I am printing some {$foo->foo}.
+Now, I am printing some {$foo
```

Comment:
this looks weird but im guessing separate issue?

Reply:
@mgrip yep, we use same logic for this as prettier use for template literals